project_task_default_stage/models/__init__.py

```diff
@@ -1,2 +1,3 @@
 # -*- coding: utf-8 -*-
+from . import project_task_type
 from . import project
```

Comment:
It would be best for the Python file to be named after the Model it implements, as originally.